### PR TITLE
Adding rust

### PR DESCRIPTION
**Pull request by willmurphyscode**
This adds support for packaging Rust executables, and examples in Rust.

### CODE REVIEW

**Comment by brendandburns** [comment]
I don't really know enough rust to review this...  I'm going to give it a try sometime soon and if it works, I will merge.

Thanks!

File: rust/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Metaparticle for Python
```
Comment (stealthybox) [inline rust/README.md]: *Rust

File: rust/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This is the implementation for Rust.
+
+## Introduction
+Metaparticle/Package simplifies and centralizes the task of
+building and deploying a container image.
+
+Here is a quick example.
+
+Consider this simple Python application:
```
Comment (stealthybox) [inline rust/README.md]: *Rust

**Comment by wagoodman** [comment]
@brendandburns let us know if there is anything else needed for this PR (it's starting to collect dust)

**Comment by brendandburns** [comment]
Sorry, I was hoping for someone with good Rust knowledge to review, but I will just merge as-is and we will go from there...

**Pull request merged**